core/streamly-core.cabal

```diff
@@ -331,15 +331,17 @@ library
                      , Streamly.Internal.Unicode.Char.Parser
                      , Streamly.Internal.Unicode.Array.Char
 
-                      -- Filesystem/IO
-                      , Streamly.FileSystem.Handle
-                      , Streamly.Internal.FileSystem.Handle
+                    -- Filesystem/IO
+                     , Streamly.FileSystem.Handle
+                     , Streamly.Internal.FileSystem.Handle
```

Comment:
This change should be done in the FileSystem.Handle related PR in which these lines are introduced.